cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -157,24 +158,24 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) {
 	fixedSomething, err := fixNodeGroupSize(autoscalingContext, time.Now())
 	if err != nil {
 		glog.Warningf("Failed to fix node group sizes: %v", err)
-		return
+		return errors.ToAutoscalerError(errors.CloudProviderError, err)
```

Comment:
If it is an error, should we log it as so? More concretely by turning `glog.Warningf` into `glog.Errorf`